packages/js/client-config-builder/src/__tests__/client-config-builder.spec.ts

```diff
@@ -107,13 +107,8 @@ describe("Client config builder", () => {
         implementations: Array.from(interfaces).map(Uri.from),
       }))
     );
-    expect(clientConfig.redirects).toStrictEqual(
-      Object.entries(testUriRedirects).map(([from, to]) => ({
-        from: Uri.from(from),
-        to: Uri.from(to),
-      }))
-    );
-    expect(clientConfig.resolvers).toStrictEqual([testUriResolver]);
+
+    // TODO: How to test resolver?
```

Comment:
We should not test that the resolver works here but that it can be added/removed and the object will be as expected. The resolver itself can be tested in the client (If I understand things correctly)

Reply:
Yup, that's my conclusion as well. However, `CoreClientConfig` only has an opaque `resolver` which cannot be tested structurally with something like `shouldEqual`.
I'm approaching this now by having the `ClientConfigBuilder` be able to "spit out" its `builderConfig` object so that its structure can be tested.